### DISCUSSION TITLE
使用docker的alist，阿里云盘open挂载的存储再挂载到webdev之后文件无法复制带本地

### DISCUSSION THREAD

**Discussion by Benjamin23333** [AlistGo/alist#3664] [Q&A]
使用docker的alist，阿里云盘open挂载的存储再挂载到webdev之后文件无法复制带本地，使用的设备是极空间z2s。
![image](https://user-images.githubusercontent.com/54323063/221878935-beb69cd9-5065-4cee-a0d7-3fe73cfd1b2e.png)
抱歉不知道如何提取日志

**Comment by rex-ll** [comment]
webDAV策略选本地代理
